### PR TITLE
Added Tumor Normal engineParameter override for tesUseInputManifest never

### LINKED ISSUE #671: Fix `wgs_tumor_normal` alignment engineParameter override for many FASTQ input files
**Context:**

Also known as "library 4 lanes split"

Case 1:
- Go to https://portal.umccr.org/subjects/SBJ00723/launch-pad/wgs-tn
- Observe amount of FASTQ files input

Case 2:
- Go to https://portal.umccr.org/subjects/SBJ00576/launch-pad/wgs-tn
- Observe amount of FASTQ files input

**Team investigation:**

See Slack thread by team on investigation:
- https://umccr.slack.com/archives/C8CG6K76W/p1716190793070379
- https://umccr.slack.com/archives/C025TLC7D/p1691624686247369?thread_ts=1691532027.452509&cid=C025TLC7D

**Issue:**

Upstream ICA WES engineParameter for TES; by default, the relevant flag set to `auto` as follows.
```
"tesUseInputManifest": "auto"
```

By changing engineParameter override to `never` should counter the issue.

```
"tesUseInputManifest": "never"
```

**Action:**

- Confirm setting the flag to "never" really solve the issue
- Improve the Portal operation QoS by fixing it

### PR DESCRIPTION
**Pull request by victorskl**
* This allows many FASTQ input files to the TN workflow!

Resolves #671

### CODE REVIEW

**Review by skanwal — APPROVED** [review]
LGTM - and +1 to Alexis point that we can enable this for all workflow types.

**Review by pdiakumis — APPROVED** [review]

File: data_processors/pipeline/domain/workflow.py [hunk 1]
```diff
@@ -244,6 +244,10 @@ def get_engine_parameters(self, target_id: str, secondary_target_id=None) -> dic
             # See https://github.com/umccr-illumina/cwl-iap/issues/200
             engine_params.update(maxScatter=8)
 
+        if self.type == WorkflowType.TUMOR_NORMAL:
```
[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/workflow.py:247]: Ah. Right. Okay, I will update it to reflect all workflow types.

[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/workflow.py]: Updated with https://github.com/umccr/data-portal-apis/commit/5e21d452e3dd5bf86dff19b59e22c61124a21732

**Review by alexiswl — APPROVED** [review]

**Comment by victorskl** [comment]
Merging...

**Pull request merged**